model/registry/exception.yaml

```
@@ -0,0 +1,50 @@
+groups:
+  - id: registry.exception
+    type: attribute_group
+    prefix: exception
+    brief: >
+      This document defines the shared attributes used to
+      report a single exception associated with a span or log.
+    attributes:
+      - id: type
+        type: string
+        brief: >
+          The type of the exception (its fully-qualified class name, if applicable).
+          The dynamic type of the exception should be preferred over the static type
+          in languages that support it.
+        examples: ["java.net.ConnectException", "OSError"]
+      - id: message
+        type: string
+        brief: The exception message.
+        examples: ["Division by zero", "Can't convert 'int' object to str implicitly"]
+      - id: stacktrace
+        type: string
+        brief: >
+          A stacktrace as a string in the natural representation for the language runtime.
+          The representation is to be determined and documented by each language SIG.
+        examples: 'Exception in thread "main" java.lang.RuntimeException: Test exception\n
+        at com.example.GenerateTrace.methodB(GenerateTrace.java:13)\n
+        at com.example.GenerateTrace.methodA(GenerateTrace.java:9)\n
+        at com.example.GenerateTrace.main(GenerateTrace.java:5)'
+      - id: escaped
+        type: boolean
+        brief: >
+          SHOULD be set to true if the exception event is recorded at a point where
+          it is known that the exception is escaping the scope of the span.
+        note: |-
+          An exception is considered to have escaped (or left) the scope of a span,
+          if that span is ended while the exception is still logically "in flight".
+          This may be actually "in flight" in some languages (e.g. if the exception
+          is passed to a Context manager's `__exit__` method in Python) but will
+          usually be caught at the point of recording the exception in most languages.
+
+          It is usually not possible to determine at the point where an exception is thrown
+          whether it will escape the scope of a span.
+          However, it is trivial to know that an exception
+          will escape, if one checks for an active exception just before ending the span,
+          as done in the [example above](#recording-an-exception).
```

```suggestion
          as done in the [example for recording span exceptions](#recording-an-exception).
```
Then it will also work in the registry markdown where it's actually below.